src/test/java/net/openhft/chronicle/bytes/Issue128Test.java

```diff
@@ -62,8 +58,13 @@ public void testCorrect() {
     public void doTest(Bytes<?> bytes, double v) {
         String format = DF.format(v);
         String output = testAppendDouble(bytes, v);
-        if (Double.parseDouble(output) != v || format.length() != output.length())
-            assertEquals(DF.format(v), output);
-//            System.out.println(DF.format(v)+" != " + output);
+        if (Double.parseDouble(output) != v || format.length() != output.length()) {
+            // Don't compare strings if we've added an exponent
+            if (!output.contains("E")) {
+                assertEquals(DF.format(v), output);
+            } else {
+                assertEquals(v, Double.parseDouble(output), 0.0);
+            }
+        }
```

Comment:
We use StringBuffer.append(..) to render doubles below some threshold (currently `6E-8`) and that writes small numbers as scientific notation. That threshold may change in the future, so I don't want to bake it into this test, but rather just continue testing all values that don't end up being rendered as scientific notation.